yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this file shouldn't be added, you can remove it by running `git rm yarn.lock`

Reply:
Corrected. Thanks